prototype/src/main/webapp/view-meeting.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Meeting</title>
+    <script src="script.js"></script>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+        text-align: left;
+      }
+    </style>
+  </head>
+  <!-- In MVP, this will just be an HTML file with empty divs where all the data goes -->
+  <!-- onload will call JS functions that fetches meeting data and renders -->
+  <body>
+    <h1>Meeting Name</h1>
+    <div id="guest-list">
+      <h2>Meeting Creator</h2>
+      <p>Elmo</p>
+      <h2>Invited Attendees</h2>
+      <ul>
+        <li>Bert</li>
+        <li>Ernie</li>
+      </ul>
+    </div>
+    <div id="meeting-time-voting">
+      <h2>Proposed Meeting Times</h2>
+      <form id="vote-time">
+        <table id="times-table">
+          <tr>
+            <th>Option</th>
+            <th>Time</th>
+            <th>Current Votes</th>
+            <th>Vote</th>
+          </tr>
+          <tr>
+            <th>1</th>
+            <th>Thursday, 14th January 10:00</th>
```

Comment:
`th` is used for headers of table rows/columns - you want `td` here (and all the other data cells in this table)

Reply:
Oh right, thank you!